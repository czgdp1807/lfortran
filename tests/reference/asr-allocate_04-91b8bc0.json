{
    "basename": "asr-allocate_04-91b8bc0",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/allocate_04.f90",
    "infile_hash": "12c9084ee1278bc669363927ae61d238c95f9e4786fd70c5797a33c6",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-allocate_04-91b8bc0.stdout",
    "stdout_hash": "530a9ce11e33c86c24ff22faab7075b5843d7d766043452d912a1aea",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}