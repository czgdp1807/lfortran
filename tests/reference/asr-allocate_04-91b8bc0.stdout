(TranslationUnit
    (SymbolTable
        1
        {
            allocate_04:
                (Program
                    (SymbolTable
                        2
                        {
                            c:
                                (Variable
                                    2
                                    c
                                    []
                                    Local
                                    ()
                                    ()
                                    Allocatable
                                    (Integer 4 [(()
                                    ())
                                    (()
                                    ())
                                    (()
                                    ())])
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            string:
                                (Variable
                                    2
                                    string
                                    []
                                    Local
                                    ()
                                    ()
                                    Allocatable
                                    (Character 1 -2 () [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    allocate_04
                    []
                    [(Allocate
                        [((Var 2 string)
                        [])]
                        ()
                        ()
                        ()
                    )
                    (Allocate
                        [((Var 2 c)
                        [((IntegerConstant 1 (Integer 4 []))
                        (IntegerConstant 3 (Integer 4 [])))
                        ((IntegerConstant 1 (Integer 4 []))
                        (IntegerConstant 3 (Integer 4 [])))
                        ((IntegerConstant 1 (Integer 4 []))
                        (IntegerConstant 3 (Integer 4 [])))])]
                        ()
                        ()
                        ()
                    )
                    (ImplicitDeallocate
                        [(Var 2 c)
                        (Var 2 string)]
                    )]
                )
        })
    []
)
